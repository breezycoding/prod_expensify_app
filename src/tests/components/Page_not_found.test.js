import React from "react";
import { shallow } from "enzyme";
import Page_not_found from "../../components/Page_not_found";

test("Should render Page_not_found", () => {
	const wrapper = shallow(<Page_not_found />);
	expect(wrapper).toMatchSnapshot();
});
	