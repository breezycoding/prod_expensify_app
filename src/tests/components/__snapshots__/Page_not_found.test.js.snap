// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Page_not_found 1`] = `
<div>
  404!!! 
  <Link
    replace={false}
    to="/"
  >
    Go Home
  </Link>
</div>
`;
